src/components/sbb-tooltip/sbb-tooltip.stories.js

```diff
@@ -0,0 +1,174 @@
+import events from './sbb-tooltip.events.ts';
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+import isChromatic from 'chromatic/isChromatic';
+import { userEvent, within } from '@storybook/testing-library';
+
+// Function to emulate pausing between interactions
+function sleep(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+// Story interaction executed after the story renders
+const playStory = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const button = canvas.getByTestId('tooltip-trigger');
+  await sleep(300);
+  await userEvent.click(button);
+};
+
+const hoverTrigger = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const showDelay = {
+  control: {
+    type: 'number',
+  },
+};
+
+const hideDelay = {
+  control: {
+    type: 'number',
+  },
+};
+
+const defaultArgTypes = {
+  'hover-trigger': hoverTrigger,
+  'show-delay': showDelay,
+  'hide-delay': hideDelay,
+};
+
+const defaultArgs = {
+  'hover-trigger': false,
+  'show-delay': undefined,
+  'hide-delay': undefined,
+};
+
+const tooltipTrigger = (position, id) => (
+  <sbb-icon
```

Comment:
I think this depends on tooltip type. With a click trigger, it should be a button, with a hover trigger it could be anything (as long as the aria attributes are set correctly).
Thoughts?

Reply:
It is what I just discussed with Marco :D

Reply:
True, I agree. The only doubt I have is, I use an `sbb-button` only with the icon for all stories except the one with the hover then?

Reply:
In theory, also hover is possible on a button :P

Reply:
I know 😄

Reply:
I know you know ;)

Reply:
I had a discussion with @mcilurzo. He will add focus and hover styles for a tooltip trigger button. My recommendation: just a global css class where a user can apply to the button. this class includes button-reset mixin and adds hover and focus-visible styles. so then we can guarantee keyboard navigation. But @kyubisation, please challenge my idea ;-)

Reply:
I will create a story with everything we discussed about the topic and the possibilities we have to implement this behavior. Thank you very much! 👍